index.bs

```diff
@@ -435,7 +435,7 @@ The <dfn attribute for=Screen>isExtended</dfn> getter steps are:
 
 The <dfn attribute for=Screen>onchange</dfn> attribute is an [=/event handler IDL attribute=] whose [=/event handler event type=] is <a event for=Screen>`change`</a>.
 
-When any [=screen/basic observable property=] of the {{Window}}'s [=/current screen=] changes, [=/Queue a global task=] on the [=/relevant global object=] of [=/this=] using the [=/window placement task source=] to [=/fire an event=] with type <dfn event for=Screen>`change`</dfn> at the {{Screen}} object referenced by {{Window}}'s {{Window/screen}} attribute.
+When any [=screen/basic observable property=] of a {{Window}} |window|'s [=/current screen=] changes, [=/queue a global task=] on the [=/relevant global object=] of |window| using the [=/window placement task source=] to [=/fire an event=] with type <dfn event for=Screen>`change`</dfn> at the {{Screen}} object referenced by |window|'s {{Window/screen}} attribute.
```

Comment:
Hey @reillyeon, this came straight from your comment on the last PR.

We can probably refine further in the future; perhaps these could even just say something like:  

```suggestion
When any attribute of a {{Screen}} |screen| changes, [=/queue a global task=] on the [=/relevant global object=] of |screen| using the [=/window placement task source=] to [=/fire an event=] with type <dfn event for=Screen>`change`</dfn> at |screen|.
```